main_model_trainer.py

```diff
@@ -0,0 +1,26 @@
+import logging
+
+from src.hotel_reservations.config import ProjectConfig
```

Comment:
You can also package your code, generate whl file and install it as a python module, then you can import from the module instead of from the relative path.

Reply:
forgot it again, done now